bundles/CoreBundle/src/Migrations/Version20230125164101.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ * @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ * @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace Pimcore\Bundle\CoreBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20230125164101 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->addSql ("DELETE FROM `users_permission_definitions` WHERE `key` = 'targeting'");
```

Comment:
https://github.com/pimcore/pimcore/pull/13781#discussion_r1065410932 Like mentioned here, when we upgrade from pimcore 10 to 11, the permission should be deleted with the table. Otherwise the bundle will not be installed, because the permission is already there

Reply:
@aryaantony92 what about the existing targeting data in the database. It says there.. right? how about mark this bundle as installed like other core bundle then you need to require and enable it in `bundles.php` on upgrade. wdyt?

Reply:
Yes, it would be good if we mark the bundles as installed and then enable in bundles.php.

Reply:
please adapt the migration accordingly. thanks!

Reply:
Done in [fbf1a13](https://github.com/pimcore/pimcore/pull/14102/commits/fbf1a135177c23f4c0412842f1e36abea5037df3) and [6147606](https://github.com/pimcore/pimcore/pull/14102/commits/6147606f08887d289d53f49560629a30263278ed)